ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -342,6 +586,17 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-boot-categories-project
+- as: nutanix-ipi-compact-image-registry-s3-f28
```

Comment:
From its name, `compact-image-registry-s3`, I don't this profile need to be covered in upgrade ci. @jianlinliu can you please help confirm?

Reply:
Hi @xiuwang do you know if we need to add upgrade job for this profile?

Reply:
@JianLi-RH Agreed with you, we don't need add upgrade for this profile.